Mathlib/NumberTheory/NumberField/CanonicalEmbedding/PolarCoord.lean

```diff
@@ -0,0 +1,528 @@
+/-
+Copyright (c) 2025 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Xavier Roblot
+-/
+import Mathlib.Analysis.SpecialFunctions.PolarCoord
+import Mathlib.NumberTheory.NumberField.CanonicalEmbedding.Basic
+
+/-!
+# Polar coordinate change of variables for the mixed space of a number field
+
+-/
+
+variable (K : Type*) [Field K]
+
+namespace NumberField.mixedEmbedding
+
+open NumberField NumberField.InfinitePlace NumberField.mixedEmbedding ENNReal MeasureTheory
+  MeasureTheory.Measure Real
+
+noncomputable section realMixedSpace
+
+/-- The real mixed space `ℝ^r₁ × (ℝ × ℝ)^r₂` with `(r₁, r₂)` the signature of `K`. --/
+abbrev realMixedSpace :=
+  ({w : InfinitePlace K // IsReal w} → ℝ) × ({w : InfinitePlace K // IsComplex w} → ℝ × ℝ)
```

Comment:
I think it should be `RealMixedSpace` since it's a `Type`?

Reply:
Well, I don't know since it is [NumberField.mixedEmbedding.mixedSpace](https://leanprover-community.github.io/mathlib4_docs/Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.html#NumberField.mixedEmbedding.mixedSpace) there. I still don't really get all the subtleties in the naming convention 😅 

Thanks for your review but please note that this PR is still WIP and I am probably going to split it into pieces to make the review easier.